distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -331,6 +331,11 @@ grant codeBase "file:/security-rest-authn-impl/security-core-api/security-core-s
     permission java.io.FilePermission "${ddf.home.perm}etc${/}org.ops4j.pax.web.cfg", "read";
 }
 
+grant codeBase "file:/org.apache.cxf.services.sts.core" {
+  permission java.io.FilePermission "${ddf.home.perm}etc${/}keystores${/}serverKeystore.jks", "read";
+  permission java.io.FilePermission "${ddf.home.perm}etc${/}keystores${/}serverTruststore.jks", "read";
+}
```

Comment:
✏️ You could reuse this grant block since every new grant block has a slight cost in terms of performance.